server/resources/translations/common.es_us.json

```diff
@@ -0,0 +1,18 @@
+{
+  "alertAcknowledged": "Otro personal de auxilio ha confirmado esta solicitud. (No necesita responder a este mensaje).",
+  "alertAdditionalAlert": "Se generó una alerta {{ alertTypeDisplayName }} adicional en {{ deviceDisplayName }}",
+  "alertCompleted": "Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
+  "alertFallback": "No se respondió a una alerta de verificación de {{ deviceDisplayName }}. Por favor, verifíquelo.",
+  "alertReminder": "Este es un recordatorio para que verifique {{ deviceDisplayName }}",
+  "alertStart": "Esta es una alerta {{ alertTypeDisplayName }}. Verifique en {{ deviceDisplayName }}. Responda con 'ok' una vez que lo haya verificado.",
+  "errorNoSession": "Error: No se encontró ninguna sesión activa",
+  "incidentCategorized": "El incidente se clasificó como {{ incidentCategory }}. Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
+  "incidentCategoryInvalid": "Lo sentimos, no se reconoció el tipo de incidente. Por favor, intente de nuevo.",
+  "incidentCategoryRequest": "Una vez que haya contestado, responda con el número que mejor describa el incidente:\n{{ incidentCategories }}",
+  "sensorDisconnectionInitial": "El Sensor Brave en {{ deviceDisplayName }} ({{ clientDisplayName }}) se ha desconectado. Presione el botón de reinicio dentro del orificio circular en la unidad de techo con un alfiler. Si no recibe un mensaje de reconexión poco después de presionar el botón de reinicio, contáctenos enviando un correo electrónico a clientsupport@brave.coop.",
+  "sensorDisconnectionReminder": "El Sensor Brave en {{ deviceDisplayName }} ({{ clientDisplayName }}) sigue desconectado. Presione el botón de reinicio dentro del orificio circular en la unidad de techo con un alfiler. Si no recibe un mensaje de reconexión poco después de presionar el botón de reinicio, contáctenos enviando un correo electrónico a clientsupport@brave.coop.",
+  "sensorIsTampered": "El sensor de la puerta en {{ deviceDisplayName }} no está completamente conectado a la puerta. Si necesita un sensor de puerta de repuesto o tiene alguna pregunta al respecto, póngase en contacto con nosotros enviando un correo electrónico a clientsupport@brave.coop.",
```

Comment:
I don't see this translation in the spreadsheet. (https://docs.google.com/spreadsheets/d/1sTUUpZFH4c4isM_pq07xNBjZvoMUtm-B/edit#gid=841616250). Where did you get it from?

Reply:
If possible, it would be great to add comments to this file to note which strings came from Google Translate and will need to be checked by a human. However, I'm not sure whether we can add comments to a JSON file. Could you please try to figure it out, but if it doesn't work, that's OK too. Thanks!